alerting/src/types/rawStormDataTypes.ts

```diff
@@ -0,0 +1,89 @@
+//TODO: Avoid duplication of code
+// the code below is duplicated from /home/max/projets/prism-app/frontend/src/context/anticipatoryAction/AAStormStateSlice/types.ts
+export enum WindState {
+  monitoring = 'monitoring',
+  ready = 'ready',
+  activated_48 = 'activated_89',
+  activated_64 = 'activated_118',
```

Comment:
This is quite weird ;)
I understand the intent but it's quite confusing. Hopefully the backend team updates the values on their side. But if not I'd rather have use use the same terms and just use them accordingly.